CHANGELOG.md

```diff
@@ -1,9 +1,17 @@
 # 2.0.0
 
-* Bumped Ion Go to `v1.1.3` allow support for unmarshalling Ion timestamps to Go time objects.
+* Bumped Ion Go dependency to `v1.1.3` allow support for unmarshalling Ion timestamps to Go time objects.
 * Logging interface now provides the LogLevel as a parameter.
```

Comment:
L4 & 5 are already listed under breaking changes, we can just remove it here.

Reply:
Done.